src/styles/nav.scss

```
@@ -0,0 +1,57 @@
+.header {
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: $max-width;
+  padding: 0 50px;
+  margin: auto;
+
+  &__logo {
+    margin: 10px 0;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__link {
+    color: $primary-color;
+    display: block;
+    align-items: center;
+    position: relative;
+    margin-left: 20px;
+    padding: 23px 0;
+    font-size: 12px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $active-color;
+    }
+
+    &.is-active::before {
+      content: " ";
+      position: absolute;
+      height: 4px;
+      width: 100%;
+      background-color: $active-color;
+      bottom: 0;
+      border-radius: 8px;
+      visibility: visible;
+    }
+  }
+
+  &__list {
+    list-style: none;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
```

move reset styles to reset.scss
```suggestion
    display: flex;
    justify-content: center;
    align-items: center;
```